### PR TITLE
Set alt name type to V_ASN1_IA5STRING

### PR DESCRIPTION
**Pull request by julek-wolfssl**
ZD15074

### CODE REVIEW

File: src/x509.c [hunk 1]
```diff
@@ -1969,11 +1969,12 @@ void* wolfSSL_X509_get_ext_d2i(const WOLFSSL_X509* x509, int nid, int* c,
                             break;
 
                         default:
-                            if (wolfSSL_ASN1_STRING_set(gn->d.ia5, dns->name,
-                                    dns->len) != WOLFSSL_SUCCESS) {
+                            if (wolfSSL_ASN1_STRING_set(gn->d.dNSName,
```
Comment (haydenroche5) [inline src/x509.c:1972]: Why was this change needed? Not mentioned in the commit message.

**Review by haydenroche5 — CHANGES_REQUESTED** [review]

**Comment by haydenroche5** [comment]
Is this in response to a ZD ticket? If so, please add the ticket number to the description.

File: src/x509.c (hunk 1 above)
Comment (julek-wolfssl) [reply src/x509.c:1972]: This was just done for consistency. It doesn't change anything in the logic.

**Comment by julek-wolfssl** [comment]
Added ticket number to description.

**Review by dgarske — APPROVED** [review]
